lib/index.js

```diff
@@ -54,20 +65,23 @@ class PWMetrics {
           results[index] = data;
         }).catch(error => {
           console.error(getMessageWithPrefix('ERROR', 'FAILED_RUN', index, results.length, error.message));
-          throw new Error(error);
+          console.error(error.stack + '\n');
```

Comment:
I put trace to help identify where the error came from. It has happened to me that an error was thrown not because of Lighthouse but because of something else, this trace allows users to understand that. It's also my own interpretation and improvement of this: https://github.com/paulirish/pwmetrics/issues/64#issuecomment-285015545. It's a minor change and I don't feel particularly attached to it.